### PR TITLE
A few accessibility fixes

### PR DESCRIPTION
**Pull request by samstradling**
I've left comments for each to explain them.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (samstradling) [inline public/index.html:6]: Not allowing zooming is bad for users who struggle to read the text and wish to zoom in to see it. This probably makes it feel less app like, but is better overall.

File: src/components/ListHeader/ListHeader.js [hunk 2]
```diff
@@ -8,10 +8,12 @@ import styles from './styles.module.css';
 const ListHeader = () => (
   <div className={styles.listHeader}>
     <div className={styles.headerRow}>
-      <Logo className={styles.logo} />
+      <Logo className={styles.logo} title="Logo" />
```
Comment (samstradling) [inline src/components/ListHeader/ListHeader.js:11]: The logo needs a title so screen readers can read that out

File: src/components/ListHeader/ListHeader.js [hunk 3]
```diff
@@ -8,10 +8,12 @@ import styles from './styles.module.css';
...
       <FontAwesomeIcon
         icon={faShareAlt}
         className={styles.shareIcon}
+        title="Share this list"
+        tabIndex={0}
```
Comment (samstradling) [inline src/components/ListHeader/ListHeader.js:16]: The title will be read out by screen readers, the tab index tells the screen reader that the item should be selected if the user is using the tab character to get through a page.

File: public/index.html (hunk 1 above)
Comment (samstradling) [reply public/index.html:6]: This brings back the zooming in mobile safari but I have a better fix for that below.

File: src/components/ShoppingList/styles.module.css [hunk 4]
```diff
@@ -34,7 +34,7 @@
 
 .newItem input {
     border-bottom: 2px solid var(--main_green);
-    font-size: 12px;
+    font-size: 16px;
```
Comment (samstradling) [inline src/components/ShoppingList/styles.module.css:37]: This is the minimum size to stop the zooming on safari, it looks a bit odd as it's now bigger than the rest of the text but we should probably increase the size of the other text too.

**Review by Joseph-Cashmore — APPROVED** [review]
Looks good, I can sort out the sizing changes once this has been merged if you like - you're right the text etc needs to be larger everywhere anyway.

**Comment by samstradling** [comment]
> Looks good, I can sort out the sizing changes once this has been merged if you like - you're right the text etc needs to be larger everywhere anyway.

That would be great, you have a much better eye for that stuff than me 😄

**Pull request merged**